serialization/src/main/scala/sbt/serialization/SerializedValue.scala

```diff
@@ -20,72 +23,59 @@ sealed trait SerializedValue {
 object SerializedValue {
   def apply[V](value: V)(implicit pickler: SPickler[V]): SerializedValue =
     JsonValue[V](value)(pickler)
+
+  /** Reconstitutes a SerialziedValue from a json string. */
+  def fromJsonString(value: String): SerializedValue =
+    JsonValue.fromJsonString(value)
+  /** Reconstitutes a SerialziedValue from a json AST. */
+  def fromJsonAST(value: JValue): SerializedValue =
+    JsonValue.fromJValue(value)
+
 }
 
+// TODO - If this is meant to handle any kind of pickle format, it needs
+//        to encode the pickle format directly.
+//        i.e. it'd be nice if we had some kind of class that had a PickleFormat and Pickle
+//        if we ever wanted to support alternative SerializedValues.
```

Comment:
We need to discuss the "responsibility" of each package, but my opinion is that the point of SerializedValue is to replace/conceal PickleFormat by restricting it to those formats we choose to support in sbt. The formats I expect we'd want to support are JSON and some compact binary format, and that's it, and you can't use anything else. `SbtPrivateSerializedValue` existed only to make toJson sbt-private, but I think we should make toJsonString public and delete both toJson and JsonValue probably. Anyhow...